discord_api.py

```diff
@@ -99,18 +92,17 @@ def writeOnFile():
 
 def help(message):
 	msg = '{0.author.mention} Aqui tens os comandos:\n'
-	msg += '!type <mensagem> - Envia uma mensagem para rodapé\n'
+	return (msg + commands_msg()).format(message)
+
+def commands_msg():
+	msg = '?type - Mostra os comandos disponíveis\n'
+	msg += '**Comandos permitidos para todos:**\n!type id - Mostra os ids das mensagens\n'
 	if author_restrict == True:
-		msg += '!type <mensagem> - Envia uma mensagem para rodapé (só os users autorizados podem adicionar mensagens)\n'
-		msg += '!type rm <id> - Remove uma mensagem de rodapé (só os users autorizados podem eliminar mensagens)\n'
-		msg += '!type id - Mostra os ids das mensagens\n'
-		msg += '!type stop - Desliga o bot (só os users autorizados podem desligar o bot)'
-	else:
-		msg += '!type <mensagem> - Envia uma mensagem para rodapé (todos os users podem adicionar mensagens)\n'
-		msg += '!type rm <id> - Remove uma mensagem de rodapé (todos os users podem eliminar mensagens)\n'
-		msg += '!type id - Mostra os ids das mensagens\n'
-		msg += '!type stop - Desliga o bot'
-	return msg.format(message)
+		msg += '**Comandos para users autorizados:**\n'
+	msg += '!type <mensagem> - Envia uma mensagem para rodapé\n'
```

Comment:
Shouldn't these all be aligned?

Like:
```python
if author_restrict == True:
        msg += '**Comandos para users autorizados:**\n'
        msg += '!type <mensagem> - Envia uma mensagem para rodapé\n'
	msg += '!type rm <id> - Remove uma mensagem de rodapé\n'
	msg += '!type stop - Desliga o bot\n'```

Reply:
No, because the last three strings are supposed to be appended to `msg`, regardless of `author_restrict` value.